app/components/views/TicketsPage/PurchaseTab/PurchaseTickets/Form.js

```diff
@@ -1,12 +1,21 @@
 import { TicketsCogs, PassphraseModalButton, ImportScriptIconButton, KeyBlueButton, InvisibleConfirmModalButton } from "buttons";
 import { AccountsSelect, NumTicketsInput } from "inputs";
 import { FormattedMessage as T } from "react-intl";
-import { TransitionMotionWrapper, ShowWarning, ExternalLink } from "shared";
+import { TransitionMotionWrapper, ShowWarning, ExternalLink, Balance } from "shared";
 
 import "style/StakePool.less";
 
 const purchaseLabel = () => <T id="purchaseTickets.purchaseBtn" m="Purchase" />;
 
+const PurchaseValidMessage = ({ spendable, numTicketsToBuy, ticketPrice }) => (
+  <div className="purchase-ticket-valid-message-area">
+    <T id="purchaseTickets.output" m="Output:" />
+    <Balance flat amount={numTicketsToBuy * ticketPrice} />
+    <T id="purchaseTickets.remaining" m="Remaining:" />
+    <Balance flat amount={spendable - numTicketsToBuy * ticketPrice} />
```

Comment:
Don't write interpolated messages like this, since different languages might want to switch the order of elements (please take a quick look at https://github.com/decred/decrediton/tree/master/app/i18n). If the messages can appear on a single line, write the following:

```js
<T 
    id="purchaseTickets.validMsg" 
    m="Output: {amount} Remaining: {remaining}" 
    values={{
         amount: <Balance flat amount={numTicketsToBuy*ticketPrice} />, 
         remaining: <Balance flat amount={spendable - numTicketsToBuy*ticketPrice} />
    }} 
/>
```

Otherwise if the design calls for two lines (one for "Output:", one for "Remaining:"), then break them into two different messages (but still using interpolated values in the message itself, instead of two different components) and style them directly with CSS (such as by adding a `purchase-ticket-valid-message-area > span` rule).

Reply:
Also I think `total` would be more appropriate than output.